config/config-example.js

```diff
@@ -136,6 +135,19 @@ Y929lRybWEiKUr+4Yw2O1W0CAwEAAQ==
 -----END PUBLIC KEY-----
 `;
 
+/**
+ * routes - where Pokemon Showdown is hosted.
+ *   Don't change this setting - there aren't any other options right now
+ */
+exports.routes = {
+	root: 'pokemonshowdown.com',
+	client: 'play.pokemonshowdown.com',
+	dex: 'dex.pokemonshowdown.com',
+	replays: 'replay.pokemonshowdown.com',
+	app: 'sim.smogon.com',
```

Comment:
Similarly, `sim.smogon.com` is explicit to Main, and should not be in an example config for side servers.